src/engine/LoadingScreen.cs

```diff
@@ -33,7 +33,6 @@ public class LoadingScreen : Control
 
     private string loadingMessage = TranslationServer.Translate("LOADING");
     private string loadingDescription = string.Empty;
-    private string tip = TranslationServer.Translate("LOADING_TIP");
```

Comment:
You should lookup the tip translation in `RandomizeTip` I think that's hooked up to be called whenever the loading screen becomes visible.

Reply:
It doen't really fit this PR, but I've implemented showing a random tip in the loading screen.
The tip is content aware, which means, that when you load into the MicrobeStage, a MicrobeStage tip is shown. When you load into the MicrobeEditor a MicrobeEditor tip is shown.
The problem is: during the longest loading steps, I don't know what I'm loading and therefore don't print a tip. The state, where I do know what I'm loading is very short, so the tip is on screen for a very short duration.
A new tip should be shown on screen after 5 seconds, but I had no way of testing this.

Reply:
Which loading screen is the one without a tooltip? Could you open an issue about that so that we remember to get that fixed at some point.

Reply:
No, you missunderstood me, sorry.
Loading consists of those steps:
`        Initial,
        ReadingData,
        ProcessingLoadedObjects,
        CreatingScene,
        Finished`
only after `ReadingData` I know what type of save I'm loading (whether I load the microbeeditor or microbestage). The problem is, that `ReadingData` is the longest step and the other steps don't last long. That is why I can't show a content aware tip most of the time.

This could be fixed by saving the `MainGameState` in the `SaveInformation`, which doesn't sound like a bad idea in the long run (for filtering, showing an icon, etc.)

Reply:
Okay, got it. That sounds nice. Adding the game state enum to SaveInformation as informatic field, would be a nice addition for the future.

Reply:
I'll create an issue.